content/java/concepts/strings/terms/toCharArray/toCharArray.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: '.toCharArray()'
+Description: 'Returns an new character array from a given string.'
+Subjects:
+  - 'Java: Strings'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+---
+
+The method `.toCharArray()` returns a new character array from the given string. Its length is equal to the the length of this string.
+
+## Syntax
+
+```java
+string.toCharArray()
+```
+- Where `string` is the string to be transform in a new array of characters.
+- This method doesn't take parameters.
+
+## Example 1
+
+Use `.toCharArray()` to print the character array `"Hello CodeCademy!"` and to demonstrate that the length is equal to the string's length:
+
+```java
+class Example {
+  public static void main(String[] args) {
+    String string = "Hello CodeCademy!";
+
+    char[] array = string.toCharArray();
+
+    System.out.println(array);
+    System.out.println("The length of the string is: "+ string.length());
+    System.out.println("The length of the array is: "+ array.length);
+  }
+}
+```
+
+The output would be:
+
+```
+Hello CodeCademy!
+The length of the string is: 17
+The length of the array is: 17
+
+```
```

Comment:
I would recommend deleting the first example. This is a pretty basic method and the additional example isn't really demonstrating some alternative functionality.